verilog/tools/ls/lsp-parse-buffer.h

```diff
@@ -90,7 +90,12 @@ class BufferTrackerContainer {
   // parsed content.
   using ChangeCallback =
       std::function<void(const std::string &uri, const BufferTracker &tracker)>;
-  void SetChangeListener(const ChangeCallback &cb) { change_listener_ = cb; }
+  void AddChangeListener(const std::string &name, const ChangeCallback &cb) {
+    change_listeners_[name] = cb;
+  }
+  void DeleteChangeListnener(const std::string &name) {
```

Comment:
We don't need `DeleteChangeListener()` currently, so maybe we should not have it until we actually need it (someone reading the code will wonder if/where it is needed if they see the funcion. So if we don't need it now we should also not necessarily have it).